apps/server/src/utils/removeFileExtension.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @description Takes a filename and removes the extension
+ * @param {string} filename - filename with extension
+ */
+export const removeFileExtension = (filename: string): string => {
+  return filename.replace(/\.[^/.]+$/, '');
+};
```

Comment:
Suggestion: how about using the `parse` method from `path` to get the filename?

If you feel there is a reason for us to prefer the regex, I would very much like to see some unit tests for it to prove that we handle most edge cases

Reply:
guess it's another thing learned today, didn't know we could remove it using `path` `parse`, I got rid of the regex